tools/autograd/derivatives.yaml

```diff
@@ -1300,18 +1300,18 @@
   self: -0.5 * grad * result.pow(3).conj()
   result: auto_element_wise
 
-- name: scatter.src(Tensor self, int dim, Tensor index, Tensor src) -> Tensor
+- name: scatter.src(Tensor self, int dim, Tensor index, Tensor src, *, bool unique_indices=False) -> Tensor
```

Comment:
Since the same indices are re-used during the backward, should this flag be propagated?

Reply:
You're right, good catch